lopper/lops/lop-domain-linux-a53-prune.dts

```diff
@@ -179,6 +180,18 @@
                                       prop_val.append(address_map[inx+2*na])
                               modify_val = update_mem_node(node1, prop_val)
                               node1['reg'].value = modify_val
+
+                              # Update symbol node referneces
+                              symbol_node = tree['/__symbols__']
+                              prop_dict = symbol_node.__props__.copy()
```

Comment:
Do you need the .copy() here ? The prop dict isn't being modified, so iterating the node properties should be enough, or am I missing something (and there is a lopper iterator that can be used to hide the internal node details).

I just want to be sure you aren't working around a lopper core bug, that I should be fixing.

Reply:
Thanks @zeddii  for review.
Here my intention is to remove the node references in the symbol node for the nodes which are deleted by this lops file, to achieve this traversing the symbol node entries against each node getting the label list and traversing the symbol node entries and deleting them if there is no match against the label list.

if we won't copy  getting the error  Exception (OrderedDict mutated during iteration).

Will fix the code part without copy

Reply:
I didn't see where that would happen on my first read through, but I can see now that the delete() being nested in the iteration of the prop_dict and the .delete() would cause that to happen.

I did of course run into this multiple times in the core of lopper, and to avoid needing a copy() I'd wrap the iterator in list()

That is a shallow copy, so it shouldn't be too expensive (and the list() isn't "free" either), but I tend to avoid copy() as inevitably I'll end up modifying the copy and not the original and wonder what broke!

I can merge this as-is, unless you want to try the list() technique to confirm if it does or doesn't work.

Reply:
Fixe with the list implementation and force pushed the commit, please review...